src/views/Register.vue

```diff
@@ -45,10 +46,18 @@ export default {
       email: "",
       password: "",
       username: "",
+      passwordConfirm: "",
+      rules: {
```

Comment:
Same as above, let's give this variable a more spefic name like "formRules"

Reply:
Good catch! I did have naming inconsistency here